src/engraving/libmscore/chordrest.cpp

```diff
@@ -949,6 +954,11 @@ EngravingItem* ChordRest::nextElement()
 
 EngravingItem* ChordRest::prevElement()
 {
+    ChordRest* prevCR = prevChordRest(this);
+    Tuplet* tuplet = prevCR ? prevCR->tuplet() : nullptr;
+    if (tuplet) {
+        return tuplet;
```

Comment:
When going backwards, you shouldn't look for tuplets on the previous ChordRest (CR). Instead you would check *current* CR. You need to know:

1. Does the current CR have a tuplet? And if so...
2. Is the current CR the first (leftmost) CR inside the tuplet?

If the answer to both questions is "yes" then you visit the tuplet, otherwise go back to the previous CR.